pkg/api/v1/types.go

```diff
@@ -1750,6 +1762,8 @@ type NodeStatus struct {
 	// Queried from cloud provider, if available.
 	// More info: http://releases.k8s.io/HEAD/docs/admin/node.md#node-addresses
 	Addresses []NodeAddress `json:"addresses,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	// Endpoints of daemons running on the Node.
+	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
```

Comment:
Since we're adding the DaemonSet API, I'm now concerned that this may be confused with that. 

How about `nodeEndpoints` or just `endpoints` (similar to `addresses`) instead?